pkgs/applications/science/misc/bada-bib/default.nix

```diff
@@ -52,7 +52,6 @@ python3Packages.buildPythonApplication rec {
   pythonPath = with python3Packages; [
     bibtexparser
     pygobject3
-    watchgod
```

Comment:
Why?

Reply:
Upstream doesn’t use watchgod anymore and instead uses the functionality that’s built-in in GTK/GLib